### PR TITLE
Develop gareth

### PR DESCRIPTION
**Pull request by BigmanRecords**

### CODE REVIEW

File: src/main/java/com/bfwg/rest/ImageController.java [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+        }
+
+        try{
+            Image image = new Image();
+
+            image.setVehicle(vehicle);
+
+            image = this.imageService.save(image);
+
+            String fileName = "image"+image.getId()+"."+file.getOriginalFilename().substring(file.getOriginalFilename().lastIndexOf(".")+1);
```
Comment (hafizhahmadzuhdi) [inline src/main/java/com/bfwg/rest/ImageController.java]: Would it be multiple file? Or we are going to allow only one file?

File: src/main/java/com/bfwg/rest/GeocodingController.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+import static org.springframework.web.bind.annotation.RequestMethod.GET;
+
+@RestController
+@RequestMapping( value="/api/geocoding", produces = MediaType.APPLICATION_JSON_VALUE)
+public class GeocodingController {
+
+    @Autowired
+    GeocodingService geocodingService;
+
+    @RequestMapping( method = GET, value="/longitude/{lon}/latidtude/{lat}")
```
Comment (hafizhahmadzuhdi) [inline src/main/java/com/bfwg/rest/GeocodingController.java]: latitude

File: src/main/java/com/bfwg/rest/ImageController.java (hunk 1 above)
Comment (BigmanRecords) [reply src/main/java/com/bfwg/rest/ImageController.java]: just leave it at one file for the upload, the relationship with vehicle to image is one to many so they can upload more images if they want

File: src/main/java/com/bfwg/rest/GeocodingController.java (hunk 2 above)
Comment (BigmanRecords) [reply src/main/java/com/bfwg/rest/GeocodingController.java]: haha typed to quickly

**Pull request merged**